<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>favorite</class>
 <widget class="QWidget" name="favorite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="nums">
         <property name="text">
          <string>收藏数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pages">
         <property name="text">
          <string>页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>从新到旧</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>从旧到新</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="jumpButton">
         <property name="text">
          <string>跳转</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>jumpButton</sender>
   <signal>clicked()</signal>
   <receiver>favorite</receiver>
   <slot>JumpPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>favorite</receiver>
   <slot>RefreshDataFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>JumpPage()</slot>
  <slot>RefreshDataFocus()</slot>
 </slots>
</ui>
